Tests/update.Tests.ps1

```diff
@@ -230,30 +264,171 @@ Describe 'prism update' {
+            "Version": "1.*"
+        },
+        {
+            "Name": "Whiskey",
+            "Version": "0.*"
+        }
+    ]
+}
+'@
+        GivenLockFile  @"
+{
+    "PSModules":  [
+        {
+            "name":  "NoOp",
+            "version":  "1.0.0",
+            "repositorySourceLocation":  "$($script:defaultLocation)"
+        },
+        {
+            "name":  "Carbon",
+            "version":  "1.0.0",
+            "repositorySourceLocation":  "$($script:defaultLocation)"
+        },
+        {
+            "name":  "Whiskey",
+            "version":  "0.14.0",
+            "repositorySourceLocation":  "$($script:defaultLocation)"
+        }
+    ]
+}
+"@
+        WhenLocking -Name ('NoOp', 'Carbon', 'Bogus')
+        # Latest NoOp version 1 should be >= 1.1.0, set at 1.0.0 for testing. This should get updated.
+        # Latest Carbon version 1 should be 1.9.0, set at 1.0.0 for testing. This should get updated.
+        # Latest Whiskey version 0 should be >= 0.61.0, set at 0.14.0 for testing. This should remain unchanged.
+        $expectedCarbonModule =
+            Find-Module -Name 'Carbon' -AllVersions | Where-Object 'Version' -like '1.*' | Select-Object -First 1
+        $carbonV0DefaultLocation =
+            Get-PSRepository -Name $expectedCarbonModule.Repository | Select-Object -ExpandProperty 'SourceLocation'
+        ThenLockFileIs ([pscustomobject]@{
+            PSModules = @(
+                [pscustomobject]@{
+                    name = 'NoOp';
+                    version = $script:latestNoOpModule.Version;
+                    repositorySourceLocation = $script:defaultLocation;
+                },
+                [pscustomobject]@{
+                    name = 'Carbon';
+                    version = $expectedCarbonModule.Version;
+                    repositorySourceLocation = $carbonV0DefaultLocation;
+                },
+                [pscustomobject]@{
+                    name = 'Whiskey';
+                    version = '0.14.0';
+                    repositorySourceLocation = $script:defaultLocation;
+                }
+            )
+        })
+    }
+
+    It 'should ignore modules that are not currently in the lock file when updating a specific set of modules' {
+        GivenPrismFile @'
+{
+    "PSModules": [
+        {
+            "Name": "NoOp",
+            "Version": "1.*"
+        },
+        {
+            "Name": "Carbon",
+            "Version": "1.*"
+        },
+        {
+            "Name": "Whiskey",
+            "Version": "0.*"
+        }
+    ]
+}
+'@
+        GivenLockFile  @"
+{
+    "PSModules":  [
+        {
+            "name":  "NoOp",
+            "version":  "1.0.0",
+            "repositorySourceLocation":  "$($script:defaultLocation)"
+        },
+        {
+            "name":  "Carbon",
+            "version":  "1.0.0",
+            "repositorySourceLocation":  "$($script:defaultLocation)"
+        }
+    ]
+}
+"@
+        WhenLocking -Name ('NoOp', 'Carbon')

```

Comment:
Does this short-hand syntax work?

```
prism update NoOp, Carbon
```

Can we add a test for it?

Reply:
Doesn't look like it. One would need to specify the -Name parameter before listing modules.

Reply:
I think all you need to do is add a [Position argument](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-5.1#position-argument) to the `Name` parameter to get this work.

Reply:
Added position properties as suggested and created a test for both commands where it calls the prism command with the shorthand syntax.

Reply:
Awesome! 👍